drawee/src/main/java/com/facebook/drawee/generic/GenericDraweeHierarchyBuilder.java

```diff
@@ -581,34 +582,36 @@ public GenericDraweeHierarchyBuilder setActualImageColorFilter(@Nullable ColorFi
    * Backgrounds are drawn in list order before the rest of the hierarchy and overlays. The
    * first background will be drawn at the bottom.
    *
+   * @deprecated use {@code setBackground} instead
    * @param backgrounds background drawables
    * @return modified instance of this builder
    */
+  @Deprecated
   public GenericDraweeHierarchyBuilder setBackgrounds(@Nullable List<Drawable> backgrounds) {
-    mBackgrounds = backgrounds;
+    if (backgrounds == null) {
+      mBackground = null;
+    } else {
+      mBackground = new ArrayDrawable(backgrounds.toArray(new Drawable[backgrounds.size()]));
+    }
     return this;
   }
 
   /**
-   * Sets a single background.
+   * Sets a background.
    *
    * @param background background drawable
    * @return modified instance of this builder
    */
   public GenericDraweeHierarchyBuilder setBackground(@Nullable Drawable background) {
-    if (background == null) {
-      mBackgrounds = null;
-    } else {
-      mBackgrounds = Arrays.asList(background);
-    }
+    mBackground = background;
     return this;
   }
 
   /**
-   * Gets the backgrounds.
+   * Gets the background.
    */
-  public @Nullable List<Drawable> getBackgrounds() {
-    return mBackgrounds;
+  public @Nullable Drawable getBackground() {
```

Comment:
This getter is in `GenericDraweeHierarchyBuilder`, and is most likely used exclusively by `GenericDraweeHierarchy` constructor. No one should really call this directly.